app/code/Magento/Eav/Model/Entity/Attribute/OptionManagement.php

```diff
@@ -6,36 +6,49 @@
 
 namespace Magento\Eav\Model\Entity\Attribute;
 
+use Magento\Eav\Api\AttributeOptionManagementInterface;
+use Magento\Eav\Api\Data\AttributeOptionInterface;
+use Magento\Eav\Model\AttributeRepository;
+use Magento\Eav\Model\ResourceModel\Entity\Attribute;
+use Magento\Eav\Model\ResourceModel\GetAttributeOptionId;
 use Magento\Framework\Exception\InputException;
 use Magento\Framework\Exception\NoSuchEntityException;
 use Magento\Framework\Exception\StateException;
 
 /**
  * Eav Option Management
  */
-class OptionManagement implements \Magento\Eav\Api\AttributeOptionManagementInterface
+class OptionManagement implements AttributeOptionManagementInterface
 {
     /**
-     * @var \Magento\Eav\Model\AttributeRepository
+     * @var AttributeRepository
      */
     protected $attributeRepository;
 
     /**
-     * @var \Magento\Eav\Model\ResourceModel\Entity\Attribute
+     * @var Attribute
      */
     protected $resourceModel;
 
     /**
-     * @param \Magento\Eav\Model\AttributeRepository $attributeRepository
-     * @param \Magento\Eav\Model\ResourceModel\Entity\Attribute $resourceModel
+     * @var GetAttributeOptionId
+     */
+    private $getAttributeOptionId;
+
+    /**
+     * @param AttributeRepository $attributeRepository
+     * @param Attribute $resourceModel
+     * @param GetAttributeOptionId $getAttributeOptionId
      * @codeCoverageIgnore
      */
     public function __construct(
-        \Magento\Eav\Model\AttributeRepository $attributeRepository,
-        \Magento\Eav\Model\ResourceModel\Entity\Attribute $resourceModel
+        AttributeRepository $attributeRepository,
+        Attribute $resourceModel,
+        GetAttributeOptionId $getAttributeOptionId
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
ok